.cargo/config.toml

```diff
@@ -32,7 +32,28 @@ documentation = """
     --no-deps
 """
 format = "fmt --all --verbose"
-lint = "clippy --workspace --all-targets --verbose -- --deny warnings -W clippy::cargo -W clippy::dbg_macro -A clippy::cargo_common_metadata -A clippy::multiple_crate_versions"
+lint = """\
+    clippy --workspace --all-targets --verbose -- \
+        --deny warnings \
+        -W clippy::cargo \
+        -A clippy::cargo_common_metadata \
+        -A clippy::multiple_crate_versions \
+        -W clippy::dbg_macro \
+        -W clippy::cast_lossless \
+        -W clippy::implicit_clone \
+        -W clippy::cloned_instead_of_copied \
+        -W clippy::copy_iterator \
+        -W clippy::explicit_into_iter_loop \
+        -W clippy::inefficient_to_string \
+        -W clippy::iter_not_returning_iterator \
+        -W clippy::manual_string_new \
+        -W clippy::needless_bitwise_bool \
+        -W clippy::needless_continue \
+        -W clippy::needless_for_each \
+        -W clippy::stable_sort_primitive \
+        -W clippy::unnecessary_join \
+        -W clippy::unnested_or_patterns \
```

Comment:
I think we can safely add these rules in `clippy.toml`, or maybe those rules can't be enabled workspace wide?

Reply:
Unfortunately it is not possible to enable rules in `clippy.toml`.

Taking a look at other repositories, [they use `rustflags`](https://github.com/web-infra-dev/rspack/blob/main/.cargo/config.toml). This seems cleaner and allow adding comments.